src/main/java/nl/tudelft/contextproject/script/Script.java

```diff
@@ -252,11 +279,30 @@ public int hashCode() {
      */
     @Override
     public Shot next() {
+        current++;
+        Shot next = shots.get(current);
+        next.execute();
```

Comment:
The camera should already be in the preset position right? So the execute is not necessary.